src/main/java/org/jenkinsci/plugins/casc/Attribute.java

```diff
@@ -167,7 +169,12 @@ public boolean equals(Owner o1, Owner o2) throws Exception {
 
     private Type _getValue(Owner target) throws Exception {
         final PropertyDescriptor property = PropertyUtils.getPropertyDescriptor(target, name);
-        if (property == null) return null;
+        if (property == null) {
+            // If this is a public final field, developers don't define getters as jelly can use them as-is
+            final Field field = FieldUtils.getField(target.getClass(), name);
```

Comment:
NoSuchFieldException will break the code, because the exception will be propagated instead of returning null

Reply:
returning null is not a correct behaviour anyway. getter is supposed to return the actual configured value